packages/grid/x-data-grid/src/hooks/features/editRows/useGridCellEditing.new.ts

```diff
@@ -151,18 +151,14 @@ export const useGridCellEditing = (
       } else if (params.isEditable) {
         let reason: GridCellEditStartReasons | undefined;
 
-        if (isPrintableKey(event.key)) {
-          if (
-            (event.ctrlKey && event.key !== 'v') ||
-            (event.metaKey && event.key !== 'v') ||
-            event.altKey
-          ) {
-            return;
-          }
+        if (isPrintableKey(event)) {
+          reason = GridCellEditStartReasons.printableKeyDown;
+        } else if (event.ctrlKey && event.key === 'v') {
```

Comment:
Related to point 4. <kbd>Cmd</kbd> + <kbd>v</kbd> is not working in macOS anymore. In https://mui.com/x/react-data-grid/editing/ it works.

```suggestion
        } else if ((event.ctrlKey || event.metaKey) && event.key === 'v') {
```

Reply:
My bad! Thanks, I should have tested it

Reply:
The "should call startCellEditMode if the paste shortcut is pressed" test only checks `ctrlKey`. If you want, you can also test when `metaKey` is pressed.